### PR TITLE
Logo in the logo

### PR DESCRIPTION
**Pull request by psuter**
It's Friday afternoon.

### CODE REVIEW

**Comment by psuter** [comment]
Should this be accepted and in case you're uncomfortable with the unicode trick, another workaround was posted in this issue today: https://github.com/spf13/cobra/issues/262 

I'm unfortunately not enough of a Go programmer to do that myself.

File: tools/go-cli/go-whisk-cli/commands/util.go [hunk 1]
```diff
@@ -427,15 +427,16 @@ func printPackageSummary(p *whisk.Package) {
...
-   \  /\  /  | | | | \__ \   <
-    \/  \/   |_| |_|_|___/_|\_\
-
-                        `
+        ____      ___                   _    _ _     _     _
+       /\   \    / _ \ _ __   ___ _ __ | |  | | |__ (_)___| | __
+  /\  /__\   \  | | | | '_ \ / _ \ '_ \| |  | | '_ \| / __| |/ /
+ /  \____ \  /  | |_| | |_) |  __/ | | | |/\| | | | | \__ \   <
+ \   \  /  \/    \___/| .__/ \___|_| |_|__/\__|_| |_|_|___/_|\_\
+  \___\/ tm           |_|
```
Comment (rabbah) [inline tools/go-cli/go-whisk-cli/commands/util.go]: tm included!

**Comment by dubeejw** [comment]
Looks good to me.

**Comment by rabbah** [comment]
Merged as https://github.com/openwhisk/openwhisk/commit/16005862a6d7b666b16224935956a93f3c0bd0fc.

**Pull request closed without merging**